fdbclient/MonitorLeader.actor.cpp

```diff
@@ -269,10 +451,18 @@ std::string ClusterConnectionString::toString() const {
 	std::string s = key.toString();
 	s += '@';
 	for (int i = 0; i < coord.size(); i++) {
-		if (i) {
+		if (_networkAddressToHostname.find(coord[i]) == _networkAddressToHostname.end()) {
+			if (i) {
```

Comment:
It's probably better to use the same `if (s.find('@') != s.length() - 1) {` check as below, in case first coord has hostname but second doesn't.